core/idl/wallet/ripple/ripple_like_wallet.djinni

```diff
@@ -105,6 +105,12 @@ RippleLikeAccount = interface +c {
     broadcastRawTransaction(transaction: binary, callback: Callback<string>);
     broadcastTransaction(transaction: RippleLikeTransaction, callback: Callback<string>);
     buildTransaction(): RippleLikeTransactionBuilder;
+    # Get fees from network
+    # Note: same note as for getFees method on BitcoinLikeAccount
```

Comment:
I’d rather have the comment duplicated instead of making an indirection (because Ripple should have nothing to do with Bitcoin).

Reply:
Segregation mode initiated ! :trollface: 👍